src/main/java/com/onarandombox/MultiversePortals/listeners/MVPPlayerMoveListener.java

```diff
@@ -104,7 +105,7 @@ public void playerMove(PlayerMoveEvent event) {
                 return;
             }
             if (!portal.isFrameValid(loc)) {
-                //event.getPlayer().sendMessage("This portal's frame is made of an " + ChatColor.RED + "incorrect material." + ChatColor.RED + " You should exit it now.");
+                p.sendMessage("This portal's frame is made of an " + ChatColor.RED + "incorrect material." + ChatColor.RED + " You should exit it now.");
```

Comment:
Could you get rid of the extra color?
```suggestion
                p.sendMessage("This portal's frame is made of an " + ChatColor.RED + "incorrect material. You should exit it now.");
```